pageserver/src/tenant/storage_layer/batch_split_writer.rs

```diff
@@ -209,6 +209,7 @@ impl<'a> SplitImageLayerWriter<'a> {
     ) -> anyhow::Result<Self> {
         Ok(Self {
             target_layer_size,
+            // XXX make this lazy like in SplitDeltaLayerWriter?
             inner: ImageLayerWriter::new(
```

Comment:
@skyzh can I nerd-snipe you into doing this? (You wrote those writer types). 

Feel free to push into this branch.

Reply:
=> https://github.com/neondatabase/neon/issues/12389